style-guide.md

```diff
@@ -0,0 +1,408 @@
+Why not? If you have a fairly complicated function, you might move that logic out into its own named function expression.
+[Source](https://github.com/airbnb/javascript#arrows--use-them).
+
+
+```
+// bad
+[1, 2, 3].map(function (x) {
+  const y = x + 1;
+  return x * y;
+});
+
+// good
+[1, 2, 3].map((x) => {
+  const y = x + 1;
+  return x * y;
+});
+```
+
+### Use === and !== over == and !=
+[Source](https://github.com/airbnb/javascript#comparison--eqeqeq).
+
+
+### Use shortcuts for booleans, but explicit comparisons for strings and numbers.
+[Source](https://github.com/airbnb/javascript#comparison--shortcuts).
+
+```
+// bad
+if (isValid === true) {
+  // ...
+}
+
+// good
+if (isValid) {
+  // ...
+}
+
+// bad
+if (name) {
+  // ...
+}
+
+// good
+if (name !== '') {
+  // ...
+}
+
+// bad
+if (collection.length) {
+  // ...
+}
+
+// good
+if (collection.length > 0) {
+  // ...
+}
+```
+
+### Ternaries should not be nested and generally be single line expressions.
+[Source](https://github.com/airbnb/javascript#comparison--nested-ternaries).
+
+```
+// bad
+const foo = maybe1 > maybe2
+  ? "bar"
+  : value1 > value2 ? "baz" : null;
+
+// split into 2 separated ternary expressions
+const maybeNull = value1 > value2 ? 'baz' : null;
+
+// better
+const foo = maybe1 > maybe2
+  ? 'bar'
+  : maybeNull;
+
+// best
+const foo = maybe1 > maybe2 ? 'bar' : maybeNull;
+
+15.7 Avoid unneeded ternary statements. eslint: no-unneeded-ternary
+// bad
+const foo = a ? a : b;
+const bar = c ? true : false;
+const baz = c ? false : true;
+
+// good
+const foo = a || b;
+const bar = !!c;
+const baz = !c;
+```
+
+### TODOs in code comments should always be accompanied by a ticket link
+That ticket should ideally include an item to remove the TODO line in the code.
+
+### Use camelCase when naming objects, functions, and instances.
+[Source](https://github.com/airbnb/javascript#naming--camelCase).
+
+```
+// bad
+const OBJEcttsssss = {};
+const this_is_my_object = {};
+function c() {}
+
+// good
+const thisIsMyObject = {};
+function thisIsMyFunction() {}
+```
+
+### Acronyms and initialisms should always be all uppercased, or all lowercased.
+Why? Names are for readability, not to appease a computer algorithm. [Source](https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms).
+
+
+```
+// bad
+import SmsContainer from './containers/SmsContainer';
+
+// bad
+const HttpRequests = [
+  // ...
+];
+
+// good
+import SMSContainer from './containers/SMSContainer';
+
+// good 
+const HTTPRequests = [ 
+  // ... 
+] ;
+
+// also good
+const httpRequests = [
+  // ...
+];
+
+// best 
+import TextMessageContainer from './containers/TextMessageContainer' ;
+
+// best
+const requests = [
+  // ...
+];
+```
+
+### You may optionally uppercase a constant only if it (1) is exported, (2) is a const (it can not be reassigned), and (3) the programmer can trust it (and its nested properties) to never change.
```

Comment:
I have to say I'm still a fairly strong 👎 on this one. I'd much prefer we didn't use them, as they call outsize attention to themselves, and could actually be misleading if used outside of convention. `const` is idiomatic and should give us all we need here.

Reply:
Sounds good. I've also given the PR the 👍 anyway, as I'm definitely open to what the group decides collectively.